### PR TITLE
go-jsonnet: init at v0.13.0

### PR DESCRIPTION
**Pull request by nshalman**
###### Motivation for this change
The go version of jsonnet is faster at compiling. It's nice to have it as an option.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/compilers/go-jsonnet/default.nix
Comment (nlewo) [inline pkgs/development/compilers/go-jsonnet/default.nix]: Could you please use `fetchFromGithub` instead.

**Review by nlewo — CHANGES_REQUESTED** [review]
You also need to add all `meta` (license, description, ...) information.

**Comment by nshalman** [comment]
> You also need to add all `meta` (license, description, ...) information.

Thanks for the review. I've updated this PR accordingly.

File: pkgs/development/compilers/go-jsonnet/default.nix
Comment (nlewo) [inline pkgs/development/compilers/go-jsonnet/default.nix]: No dot at the end

File: pkgs/development/compilers/go-jsonnet/default.nix
Comment (nlewo) [inline pkgs/development/compilers/go-jsonnet/default.nix]: Would be better to use a tag:
```rev = v${version};```

**Review by nlewo — CHANGES_REQUESTED** [review]
Would you like to become the maintainer of this package? In this case, you could add yourself in the maintainer list.

File: pkgs/development/compilers/go-jsonnet/default.nix
Comment (nshalman) [reply pkgs/development/compilers/go-jsonnet/default.nix]: I looked at that, but the current version tag is on the wrong commit on the upstream repo (I don't know why.)

File: pkgs/development/compilers/go-jsonnet/default.nix
Comment (nshalman) [reply pkgs/development/compilers/go-jsonnet/default.nix]: Fixed.

**Comment by nshalman** [comment]
> Would you like to become the maintainer of this package? In this case, you could add yourself in the maintainer list.

Only if I have to...

File: pkgs/development/compilers/go-jsonnet/default.nix
Comment (nlewo) [reply pkgs/development/compilers/go-jsonnet/default.nix]: Hmm, actually, you are not fetching the release `0.12.1` but a non released version.
So, could you fetch a release by using the tag?

**Review by nlewo — CHANGES_REQUESTED** [review]

**Comment by nlewo** [comment]
It's not mandatory for a package to have a maintainer but it's really better! If you are the maintainer, it is easier for you to do upgrades because we merge more quickly upgrades from maintainers. You are also notified on upgrades.
So, it's up to you ;)

**Comment by nshalman** [comment]
I'm going to wait for upstream to cut a new release and point to that tag. I'll add myself as a maintainer at the same time.

**Comment by nshalman** [comment]
@nlewo Upstream has tagged a new release, and I've updated this PR accordingly. Please review again when you have a moment.

File: pkgs/development/compilers/go-jsonnet/default.nix
Comment (nshalman) [reply pkgs/development/compilers/go-jsonnet/default.nix]: Whoops, this is applicable now. Need to tweak once more.

**Comment by nlewo** [comment]
@GrahamcOfBorg build go-jsonnet

**Comment by nlewo** [comment]
Thx!

**Pull request merged**
